src/Common/src/Interop/Windows/user32/Interop.EnumWindows.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        internal delegate bool EnumThreadWindowsCallback(IntPtr hWnd, IntPtr lParam);
+
+        [DllImport(Libraries.User32)]
```

Comment:
OK, but which should we use in our Pinvoke declarations -- API set or classic name? I know that both work, but presumably going via the API set is more robust.